gc3apps/ivs.uzh/gtree.py

```diff
@@ -0,0 +1,211 @@
+#! /usr/bin/env python
+#
+#   gtree.py -- Front-end script for running an R script
+#   over a defined Data set on a given number of nodes .
+#
+#   Copyright (C) 2014, 2015 GC3, University of Zurich
+#
+#   This program is free software: you can redistribute it and/or
+#   modify
+#   it under the terms of the GNU General Public License as published by
+#   the Free Software Foundation, either version 3 of the License, or
+#   (at your option) any later version.
+#
+#   This program is distributed in the hope that it will be useful,
+#   but WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#   GNU General Public License for more details.
+#
+#   You should have received a copy of the GNU General Public License
+#   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+
+It uses the generic `gc3libs.cmdline.SessionBasedScript` framework.
+
+See the output of ``gtree.py --help`` for program usage
+instructions.
+
+Input parameters consists of:
```

Comment:
Please do not submit unfinished docstrings in a PR.  If you think it's not relevant for the final result, just remove it.